src/test/java/step_definitions/SalesForceMarketing.java

```diff
@@ -128,6 +129,61 @@ public void click_contactTab() throws InterruptedException {
 
     }
 
+    @Then("user selects the filter")
+    public void selectFilter() throws InterruptedException {
+        CampaignUtil.selectFilterInContacts(dataMap.get("FieldValue"),dataMap.get("ContactLastName"));
+        KeywordUtil.takeScreenshotAndAttachInReport();
+    }
+
+    @Then("user edit the contact")
+    public void edit_Contact() throws InterruptedException {
+        CampaignUtil.editContact(KeywordUtil.generateRandomString(4) + "@gmail.com",dataMap.get("ContactLastName"));
+        CampaignUtil.verificationMessage();
+        KeywordUtil.takeScreenshotAndAttachInReport();
+    }
+
+    @Then("user Delete the contact")
+    public void deleteContact() throws InterruptedException {
+       CampaignUtil.deleteContact(dataMap.get("ContactLastName"));
+       KeywordUtil.takeScreenshotAndAttachInReport();
+    }
+
+    @Then("user remove the filter")
+    public void removeFilter() throws InterruptedException {
+         CampaignUtil.removeFilter();
+    }
+
+    @Then("user create new contact in Contact")
+    public void createNewContact() throws InterruptedException {
+        CampaignUtil.createConatctInConatctTab( dataMap.get("ContactLastName"), KeywordUtil.generateRandomString(4) + "@gmail.com");
+        CampaignUtil.verificationMessage();
+        KeywordUtil.takeScreenshotAndAttachInReport();
+    }
+
+    @Then("user create new opportunity in Contact")
+    public void createOpportunityInConatct() throws InterruptedException {
```

Comment:
Please always check the spelling, contact spelling is not correct, review your code 2-3 times before raising PR